app/build.gradle.kts

```diff
@@ -140,6 +141,9 @@ dependencies {
     implementation(platform("com.squareup.okhttp3:okhttp-bom:4.12.0"))
     implementation("com.squareup.okhttp3:okhttp")
     implementation("com.squareup.okhttp3:logging-interceptor")
+
+    //QR CODE
+    implementation ("com.google.zxing:core:3.4.1")
 }
```

Comment:
this shouldn't be in this PR (only in the one you made for the QR code generation)

Reply:
sorry, accidental, I will swap